components/modal/modal.js

```diff
@@ -200,6 +204,7 @@ Modal.defaultProps = {
   onClose: null,
   onOverlayClick: null,
   overlay: true,
+  size: false,
```

Comment:
Passing `false` is odd here, isn't it? Wouldn't `null` be a better default?

Reply:
Also, where is this being used?

Reply:
There are two sizes of modal available: big (max-width: 780), small (max-width: 480).
If I specify size as Boolean I prefer set it as *false* instead of *null* value.

Reply:
I guess here the prop should be `isSmall` and then will have the default `false`

Reply:
yep, thanks, fixing